pypeit/images/rawimage.py

```diff
@@ -39,7 +39,7 @@ def __init__(self, filename, spectrograph, det):
         self.det = det
 
         # Load the raw image and the other items of interest
-        self.raw_image, self.hdu, self.exptime, self.rawdatasec_img, self.oscansec_img = self.spectrograph.get_rawimage(
+        self.raw_image, self.headarr, self.exptime, self.rawdatasec_img, self.oscansec_img = self.spectrograph.get_rawimage(
```

Comment:
We didn't get a chance to discuss at the PypeIt meeting, but I'd much prefer we stick with `self.hdu`.  You can construct `headarr` from it wherever you need it from `spectrograph.get_headarr(hdu)`.  

Also, I need `hdu` for the `detector` as coded in a upcoming branch.

Do you mind putting this back?  If not, I probably will need to in my branch..

Reply:
Done!